editor/src/messages/portfolio/document/node_graph/graph_operation_message_handler.rs

```diff
@@ -689,6 +695,8 @@ impl MessageHandler<GraphOperationMessage, GraphOperationHandlerData<'_>> for Gr
 						})
 						.unwrap_or_default();
 
+					modify_inputs.document_network.nodes.get_mut(&id).unwrap().alias = alias.clone();
```

Comment:
Please avoid this unwrap, perhaps using `if let`.

Reply:
I made this change, but please look at the code to learn from the updated code for your future reference.